recipes/nbappinator/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "nbappinator" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nbappinator-{{ version }}.tar.gz
+  sha256: 021ad8dc6a4f8c55562e95891dc67708e40fa639f85c9d60d6864f58d77aae14
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - ipywidgets >=7.8.0,<7.9.dev0
+    - plotly
+    - ipyvuetify >=1.9
+    - ipyaggrid
+    - ipytree
+    - pandas >=1.5.3
+
+test:
+  imports:
+    - nbappinator
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/iqmo-org/nbappinator
+  dev_url: https://github.com/iqmo-org/nbappinator
```

Comment:
This URL is returning 404.

Reply:
Oh, thank you for pointing that out. Just changed to Public now.